### PR TITLE
Daily motivation

### PR DESCRIPTION
**Pull request by pradeepthipani**
Added a new code in Parsers folder: Daily motivation.

The bot sends a random motivational quote when the user types !motivate.

### CODE REVIEW

**Comment by pradeepthipani** [comment]
Sorry, That was my bad. I have renamed it now please validate.

**Comment by wiz0floyd** [comment]
This would be better if it had more than just 5 quotes. Perhaps call an API that returns motivational responses.

File: Parsers/Daily Motivation.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+order:100
+stop_processing:false
+*/
+
+var quotes = [ //Add more number of quotes based on need
+    "The only way to do great work is to love what you do. – Steve Jobs",
+    "Success is not the key to happiness. Happiness is the key to success. – Albert Schweitzer",
+    "Believe you can and you're halfway there. – Theodore Roosevelt",
+    "The future belongs to those who believe in the beauty of their dreams. – Eleanor Roosevelt"
+];
```
Comment (wiz0floyd) [inline Parsers/Daily Motivation.js:14]: Add more than 4 quotes or call an API to get quotes.

Comment (pradeepthipani) [reply Parsers/Daily Motivation.js]: > Add more than 4 quotes or call an API to get quotes.

I have updated the code please review it.

**Review by earlduque — APPROVED** [review]

**Pull request merged**